--- fdutils-5.4/configure.in~	Thu Jul  1 19:22:51 1999
+++ fdutils-5.4/configure.in	Wed Aug 15 21:06:02 2001
@@ -20,8 +20,6 @@
 
 AC_CHECK_HEADERS(linux/ext_fs.h linux/xia_fs.h)
 
-dnl Make fdmount accessible by the floppy group only
-undefine([fdmount-floppy-only])dnl
 AC_ARG_ENABLE(fdmount-floppy-only,
 [  --enable-fdmount-floppy-only       Make fdmount accessible to the "floppy" group only],
 [if test x$enableval = xyes; then
